drivers/azure/azureutil/azureutil.go

```diff
@@ -159,6 +160,9 @@ func (a AzureClient) CreatePublicIPAddress(ctx *DeploymentContext, resourceGroup
 			Location: to.StringPtr(location),
 			Properties: &network.PublicIPAddressPropertiesFormat{
 				PublicIPAllocationMethod: ipType,
+				DNSSettings: &network.PublicIPAddressDNSSettings{
+					DomainNameLabel: to.StringPtr(name),
```

Comment:
Long story. The ConfigureWinRM.PS1 script needed FQDN, which is used as the CN in the cert. Later I realized that I can make the client ignore the CN, so this is not needed. Will remove.